src/com/nkita/practice/collections/PersonStatisticHandler.java

```diff
@@ -219,4 +219,84 @@ public void checkOldestPerson(Gender gender) {
             System.out.println("The oldest Person(s) witch gender " + gender + "\n" + oldestPersons);
         }
     }
-}
\ No newline at end of file
+
+    public void checkSortedStatistic() {
+        List<Person> personList = new ArrayList<>(persons);
+        personList.sort(new Comparator<Person>() {
+            @Override
+            public int compare(Person age1, Person age2) {
+                if (age1.getAge() > age2.getAge()) {
+                    return -1;
+                } else if (age1.getAge() < age2.getAge()) {
+                    return 1;
+                } else {
+                    return 0;
+                }
+
+            }
+        });
+        Collections.sort(personList);
+
+        for (Person o : personList) {
+            System.out.println(o.getName() + " | " + o.getAge());
+        }
+    }
+
+
+    public void checkFilteredStatistic(PersonFilterRequest personFilterRequest) {
+        List<Person> personList = new ArrayList<>();
+        for (Person person : persons) {
+
+            if (isPersonValidForFilters(personFilterRequest.getPersonFilters(), person, personFilterRequest)) {
+                personList.add(person);
+            }
+        }
+        printInfo(personList);
+    }
+
+    private boolean isPersonValidForFilters(List<PersonFilter> personFilters, Person person, PersonFilterRequest personFilterRequest) {
```

Comment:
'List <PersonFilter>  personFilters' is useless parameter here because
"PersonFilterRequest personFilterRequest" already contain person filters, and you can get these filters from it.